lib/Donors/utils.js

```diff
@@ -12,7 +12,7 @@ const getDonorUrl = (orgId) => {
   return undefined;
 };
 
-export const getResultsFormatter = ({
+export const getDonorsListFormatter = ({
```

Comment:
This fn is default formatter for donors list, you don't need https://github.com/folio-org/stripes-acq-components/pull/726/commits/861fd1f5e9e1305cec4b83c739dd28fdebe6b1fc#diff-a271f2982976d9361567a1e9a93f59d0389505adfa4ce5aa9bef08f0b96333c7R15

It should format `name` and `code`, and should be used in donors list by default.
For the form you should extend formatter with `unassignDonor`